plugins/base/src/test/kotlin/renderers/html/DivergentTest.kt

```diff
@@ -320,11 +324,8 @@ class DivergentTest : RenderingOnlyTestBase() {
 
         HtmlRenderer(context).render(page)
         renderedContent.match(
-            "a-",
-            Div(Div(Div("a")), Div(Div("NATIVE"))),
-            "ab+b-",
-            Div(Div(Div("b")), Div(Div("NATIVE"))),
-            "ab+"
+            Div("a-", Div(Div(Div("a")), Div(Div("NATIVE"))), "ab+"),
```

Comment:
@kamilok1965 could you look at those lines? I think they are correct and previous tests were not wrapped in additional `div` so they merged into one like strings

Reply:
I agree, it looks better now